packages/common/src/project/readers/ipfs-reader.ts

```diff
@@ -28,28 +31,26 @@ export class IPFSReader implements Reader {
     return undefined;
   }
 
-  async getPkg(): Promise<IPackageJson | undefined> {
-    return Promise.resolve(undefined);
+  // eslint-disable-next-line @typescript-eslint/require-await
+  async getPkg(): Promise<IPackageJson> {
+    throw new Error('Project package.json is not found');
```

Comment:
This is a change in behaviour, we need to check that nowhere calls this
```suggestion
    throw new Error('IPFS Reader doesn't support reading package.json');
```